HWweek6.py

```diff
@@ -0,0 +1,87 @@
+class TreeNode(object):
+    def __init__(self, x):
+        self.val = x
+        self.left = None
+        self.right = None
+
+def isequal(rootS, rootT):
+    if rootS is None and rootT is None:
+        return True
+    if rootS is None or rootT is None:
+        return False
+    return rootS.val == rootT.val and isequal(rootS.left, rootT.left) and isequal(rootS.right, rootT.right)
+    
+def isSubtree(s, t):
+    if s is None:
+        return False
+    if t is None:
+        return False
+    if (isequal(s, t)):
+        return True
+    return isSubtree(s.left, t) or isSubtree(s.right, t)
+
+#Test 1
```

Comment:
Thanks for uploading your written TEBOW IT notes! 

You may already be doing this, but make sure that you practice writing our your implementation via pen and paper/whiteboard before you code it up on a computer. It's closer to what you'll be doing for on-campus interviews, and let's you know if there's a difference in your writing/typing speeds (I type much faster than I write, for example, so writing out the answers gives me a more accurate picture of how long the problem would take me in an interview).